internal/resource/configmap.go

```diff
@@ -137,3 +134,11 @@ func AppendIfUnique(a []string, b []rabbitmqv1beta1.Plugin) []string {
 	}
 	return set
 }
+
+func updateProperty(configMapData map[string]string, key string, value string) {
```

Comment:
Can use a better name. Any suggestion @coro?

Reply:
Can't think of anything better.

Reply:
Let's merge it then.